community/releases/release-manager-runbook/index.md

```diff
@@ -339,3 +379,16 @@ Repeat as needed.
 
 Follow the instructions in deck-kayenta’s
 [README](https://github.com/spinnaker/deck-kayenta#publishing-spinnakerkayenta).
+
+
+## Troubleshooting
+
+### Flow_BuildAndValidate_* is getting 409s when trying to upload to Bintry
```

Comment:
I'd prefer that buildcop-related troubleshooting not be duplicated here since we already have a section on resolving Bintray 409s: https://spinnaker.io/community/contributing/nightly-builds/#bintray-conflicts

Reply:
oh cool, i didn't know about that! I added a note to the top of the runbook!